### PR TITLE
updated full console and facebook + google login tests

### PR DESCRIPTION
**Pull request by freglyc**
Current Safari issue: facebook login method fails

### CODE REVIEW

File: pages/accounts/home.py [hunk 1]
```diff
@@ -56,6 +69,44 @@ def login(self, user, password):
             self.find_element(*self._login_submit_button_locator).click()
             self.wait.until(lambda _: self.logged_in)
 
+        def facebook_login(self, user, facebook_user, password):
```
Comment (gregfitch) [inline pages/accounts/home.py]: Check through these sleeps...Would they make more sense to use a wait (`self.wait.until(...)`) before interacting with the elements?

Comment (freglyc) [reply pages/accounts/home.py]: Updated!

File: tests/accounts/test_accounts_profile.py [hunk 2]
```diff
@@ -132,19 +132,31 @@ def test_add_a_verified_email(accounts_base_url, selenium):
 
 
 @test_case('C195555')
-@expected_failure
 @accounts
 @social
 def test_profile_login_using_google(accounts_base_url, selenium):
     """Test the Gmail login method."""
+    page = Profile(selenium, accounts_base_url).open()
```
Comment (m1yag1) [inline tests/accounts/test_accounts_profile.py]: Tests should use Given, When, Then. This is related to issue #12

File: tests/accounts/test_accounts_profile.py [hunk 3]
```diff
@@ -173,12 +183,22 @@ def test_admin_pop_up_console(accounts_base_url, selenium):
 @accounts
 def test_go_to_full_console(accounts_base_url, selenium):
```
Comment (gregfitch) [inline tests/accounts/test_accounts_profile.py]: For this test, the `GIVEN` should be a logged in administrator; we're evaluating the full console link, not the log in process.

**Pull request merged**